src/common.ts

```diff
@@ -14,4 +40,12 @@ export interface CreateSessionOptions {
 }
 export type GetTimestamp = {
   nanos: number; seconds: number;
-};
+}
+export interface TransactionOptions {
```

Comment:
Pretty sure this object was superseded by `TimestampBounds` and we shouldn't need it, or at the very least we should remove the `timeout` option.